tests/KdybyTests/Events/mocks.php

```diff
@@ -165,7 +165,7 @@ class EventArgsMock extends Kdyby\Events\EventArgs
 /**
  * @author Filip Procházka <filip@prochazka.su>
  */
-class EventListenerMock extends Nette\Object implements Kdyby\Events\Subscriber
+class EventListenerMock implements Kdyby\Events\Subscriber
```

Comment:
Because of `Nette\Object__call` implementation, `is_callable` returned `TRUE` and tests has been failing after `callable` support.